hyperspy/tests/model/test_eelsmodel.py

```diff
@@ -190,6 +198,195 @@ def test_remove_components(self):
         self.m.remove(comp)
         assert len(self.m) == 2
 
+    def test_fit_wrong_kind(self):
+        with pytest.raises(ValueError):
+            self.m.fit(kind="wrongkind")
+
+    def test_enable_background(self):
+        self.m.components.PowerLaw.active = False
+        self.m.enable_background()
+        assert self.m.components.PowerLaw.active
+
+    def test_disable_background(self):
+        self.m.components.PowerLaw.active = True
+        self.m.disable_background()
+        assert not self.m.components.PowerLaw.active
+
+    def test_signal1d_property(self):
+        assert self.s == self.m.signal1D
+        s_new = hs.signals.EELSSpectrum(np.ones(200))
+        s_new.set_microscope_parameters(100, 10, 10)
+        self.m.signal1D = s_new
+        assert self.m.signal1D == s_new
+
+    def test_signal1d_property_wrong_value_setter(self):
+        m = self.m
+        s = hs.signals.Signal1D(np.ones(200))
+        with pytest.raises(ValueError):
+            self.m.signal1D = s
+
+    def test_remove(self):
+        m = self.m
+        c_k = m.components.C_K
+        assert c_k in m
+        m.remove(c_k)
+        assert not (c_k in m)
+
+    def test_quantify(self):
+        self.m.quantify()
```

Comment:
Perhaps an outcome of the result of this call should be tested?

Reply:
Thanks for also showing me a way to catch output I didn't know of before!

Reply:
It was a bit tricky, since `quantify` uses `print`. So I had to "catch" the stdout.

But it seems to work :)